lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1385,13 +1392,23 @@ bool TypeSystemSwiftTypeRef::IsVoidType(opaque_compiler_type_t type) {
   };
   VALIDATE_AND_RETURN(impl, IsVoidType, type, (ReconstructType(type)));
 }
-// Type Completion
-bool TypeSystemSwiftTypeRef::GetCompleteType(opaque_compiler_type_t type) {
-  return m_swift_ast_context->GetCompleteType(ReconstructType(type));
-}
 // AST related queries
 uint32_t TypeSystemSwiftTypeRef::GetPointerByteSize() {
-  return m_swift_ast_context->GetPointerByteSize();
+  auto impl = [&]() -> uint32_t {
+    if (auto *module = GetModule()) {
+      auto &triple = module->GetArchitecture().GetTriple();
+      if (triple.isArch64Bit())
+        return 8;
+      if (triple.isArch32Bit())
+        return 4;
+      if (triple.isArch16Bit())
+        return 2;
```

Comment:
llvm_unreachable?

Reply:
I don't feel comfortable to crash LLDB under a condition that can appear due to bad/unexpected input, such as an unrecognized triple.